main.go

```diff
@@ -56,6 +56,12 @@ func main() {
 			Usage:       "Key file for etcd connection",
 			Destination: &config.ServerTLSConfig.KeyFile,
 		},
+		&cli.BoolFlag{
```

Comment:
nit: can you move this up, directly below the `Destination: &config.BackendTLSConfig.CertFile` flag? We could probably move the `Destination: &config.BackendTLSConfig.KeyFile` flag up as well, just to keep all the backend flags together.